assembly/config.json

```diff
@@ -37,7 +37,7 @@
         {
             "org": "temporalio",
             "name": "documentation-samples-go",
-            "ref": "add-go-schedule-sample"
+            "ref": "sync_update"
```

Comment:
I see "sync update" in a few places in our naming of samples and such here. I wonder if we should just call these "update" (as there is a concept of async update).